sdk/identity/identity/src/index.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { TokenCredential } from '@azure/core-http';
+import { EnvironmentCredential } from './credentials/environmentCredential'
+
+export { AccessToken } from './credentials/accessToken'
+export { AzureCredential } from './credentials/azureCredential'
```

Comment:
because we moved "default" off of AzureCredentials it is really just an implementation detail and shouldn't be exported.

Reply:
Good point, I'll remove it from public exports.